pyiron_atomistics/atomistics/master/quasi.py

```diff
@@ -173,3 +173,60 @@ def optimise_volume(self, bulk_eng):
             entropy_lst.append(entropy[ind])
             cv_lst.append(cv[ind])
         return v0_lst, free_eng_lst, entropy_lst, cv_lst
+
+    def plot_free_energy_volume_temperature(self,
+                                            murnaghan_job,
+                                            temperature_start=None,
+                                            temperature_end=None,
+                                            temperature_steps=None):
+        """
+        Plot volume vs free energy curves for defined temperatures.
+
+        Args:
+            murnaghan_job: job_name or job_id of the Murnaghan job
+            temperature_start: if None, value will be used from job.input['temperature_start']
+            temperature_end: if None, value will be used from job.input['temperature_end']
+            temperature_steps: if None, value will be used from job.input['temperature_steps']
+
+        Returns:
+            Volume vs free energy curves for defined temperatures.
```

Comment:
The method is not actually returning anything so this should be blank.  It might make sense to return the current axis and allow passing one in, as in e.g. the [phonopy](https://github.com/pyiron/pyiron_atomistics/blob/4ab5279c2096ae0859aaf55f8eb20f35b5a0cf13/pyiron_atomistics/atomistics/master/phonopy.py#L499).

Reply:
current axis returned.